.gitignore

```diff
@@ -1,3 +1,45 @@
+# phpstorm project files
```

Comment:
These are usually IDE specific file and are better placed in a global.gitignore file rather than in the project file. As a policy for repositories I manage, I won't be merging IDE specific files in since they vary so much between IDEs.

Reply:
There are only 3 major IDEs used for PHP development last decade. This list doesn't change often. I can remove these lines, if you want.